public/kbn_sankey_vis_controller.js

```diff
@@ -45,106 +45,104 @@ module.controller('KbnSankeyVisController', function ($scope, $element, $rootSco
   };
 
   let _buildVis = function (data) {
+
     if(!resize){
       data.slices=filterNodesAndLinks(data.slices.nodes, data.slices.links);
     }
     $scope.emptyGraph = (data.slices.nodes.length <= 0) ;
 
-    _updateDimensions();
-
     let energy = data.slices;
     div = d3.select(svgRoot);
     if (!energy.nodes.length) return;
 
     svg = div.append('svg')
-      .attr('width', width)
-      .attr('height', height + margin)
-      .append('g')
-      .attr('transform', 'translate(0, 0)');
+        .attr('width', width)
```

Comment:
Why did you increase indentation here (and at some places below) from one to two spaces? Isn't two spaces correct in terms of our coding conventions?

Reply:
This was actually a mistake made due the IDE when formatting  the code.
It will not occur in the future releases.

Reply:
The indentations were modified now to follow the Kibana's conventions.